### PR TITLE
[MRG] fr-translation

### PR DESCRIPTION
**Pull request by michelfernandez**
- translation fr for all pages
- identification factsheets for FR are in translation, but currently integrated in EN

### CODE REVIEW

File: site/content/french/contact.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Skribble – Kontakt
```
Comment (fursinger) [inline site/content/french/contact.md]: We should write french meta title and description tags.

File: site/content/french/services.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+{{% /plan %}}
+{{% plan purple "Identification facile" "Nous identifions les signataires" %}}
+Convient si vous utilisez la QES avec un nombre de personnes prédéfini.
+{{% /plan %}}
+{{< /row >}}
+{{% /content %}}
+
+{{% cta
+  "Plus d'informations sur l'identification"
+  "/identification"
```
Comment (fursinger) [inline site/content/french/services.md]: should be "/fr/identification"

File: site/content/french/contact.md (hunk 1 above)
Comment (michelfernandez) [reply site/content/french/contact.md]: Thank you, I've adapted!

File: site/content/french/services.md (hunk 2 above)
Comment (michelfernandez) [reply site/content/french/services.md]: done!

**Pull request merged**